src/functions-reference/positive_lower-bounded_distributions.qmd

```diff
@@ -303,6 +346,61 @@ of the non-decision time `var_tau`, dropping constant additive terms.
 Setting `var_delta`, `var_beta`, and `var_tau` to `0` recovers the 4-parameter signature above.
 {{< since 2.35 >}}
 
+---
+
+**Note:** The `lcdf` and `lccdf` functions for the `wiener` distribution are
+implemented defectively, meaning that the functions do **not** integrate to $1$,
+but rather to the probability to hit [one or the other response boundary](#boundaries).
+
+
+<!-- real; wiener_lcdf_defective; (real y, real alpha, real tau, real beta, real delta); -->
+\index{{\tt \bfseries wiener\_lcdf\_defective }!{\tt (real y, real alpha, real tau, real beta, real delta): real}|hyperpage}
+
+`real` **`wiener_lcdf_defective`**`(real y, real alpha, real tau, real beta, real delta)`<br>\newline
+
+The log of the cumulative distribution function (CDF) of the Wiener distribution
+of `y` given boundary separation `alpha`, non-decision time `tau`, a-priori bias
+`beta`, and drift rate `delta`.
+
+{{< since 2.38 >}}
+
+<!-- real; wiener_lccdf_defective; (real y, real alpha, real tau, real beta, real delta); -->
+\index{{\tt \bfseries wiener\_lccdf\_defective }!{\tt (real y, real alpha, real tau, real beta, real delta): real}|hyperpage}
+
+`real` **`wiener_lccdf_defective`**`(real y, real alpha, real tau, real beta, real delta)`<br>\newline
+
+The log of the complementary cumulative distribution function (CDF) of the
+Wiener distribution of `y` given boundary separation `alpha`, non-decision time
+`tau`, a-priori bias `beta`, and drift rate `delta`.
+
+{{< since 2.38 >}}
+
+<!-- real; wiener_lcdf_defective; (real y, real alpha, real tau, real beta, real delta, real var_delta, real var_beta, real var_tau); -->
+\index{{\tt \bfseries wiener\_lcdf\_defective }!{\tt (real y, real alpha, real tau, real beta, real delta, real var\_delta, real var\_beta, real var\_tau): real}|hyperpage}
+
+`real` **`wiener_lcdf_defective`**`(real y, real alpha, real tau, real beta, real delta, real var_delta, real var_beta, real var_tau)`<br>\newline
+
+The log of the cumulative distribution function (CDF) of the Wiener distribution
+of `y` given boundary separation `alpha`, non-decision time `tau`, a-priori bias
+`beta`, drift rate `delta`, inter-trial drift rate variability `var_delta`,
+inter-trial variability of the starting point (bias) `var_beta`, and inter-trial
+variability of the non-decision time `var_tau`.
+
```

Comment:
Comment:  Now that I'm reading this, the three extra arguments are becoming more clear to me.  It looks like this implementation is marginalizing out the actual values.  The alternative is to use these variances to draw the `delta`, `beta`, and `tau` parameters, then plug them into the four-parameter model.